source/uwp/Renderer/AdaptiveCardRenderer.vcxproj

```diff
@@ -120,7 +120,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup>
     <ClCompile>
-      <PreprocessorDefinitions>_WINRT_DLL;_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_WINRT_DLL;_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING;_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
This vcpxroj statically links ObjectModelProjection.vcxproj.  That vcxproj does not define `_DISABLE_CONSTEXPR_MUTEX_CONSTRUCTOR`.  Is that going to create ODR violations?

Reply:
updated it